### PR TITLE
Issue #18497 Update URL Request and Hello World to Japanese

### PR DESCRIPTION
**Pull request by SupriyaSirbi**
Update 'Received request for URL: ' + request.url and 'Hello World!' to be in Japanese, on https://kubernetes.io/ja/examples/minikube/server.js.

### CODE REVIEW

**Comment by inductor** [comment]
@SupriyaSirbi I'mafraid I don't think this is necessary.

**Comment by SupriyaSirbi** [comment]
> @SupriyaSirbi I'm afraid I don't think this is necessary.
> Could you tell us why you need this?

Hello,

I just worked on this issue and was not aware that this is not necessary. I shall close the PR if examples need not be translated. The Translation is available in Russian Language so thought this might be needed.

Thanks,
Supriya

File: content/ja/examples/minikube/server.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 var http = require('http');
 
 var handleRequest = function(request, response) {
-  console.log('Received request for URL: ' + request.url);
+  console.log('URLのリクエストを受信しました ' + request.url);
   response.writeHead(200);
-  response.end('Hello World!');
+  response.end('こんにちは世界');
```
Comment (inductor) [inline content/ja/examples/minikube/server.js]: Especially this change is too much while the other change would be "okay". 

"Hello World" is enough popular and known concept for any programmers in Japan.

File: content/ja/examples/minikube/server.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
 var http = require('http');
 
 var handleRequest = function(request, response) {
-  console.log('Received request for URL: ' + request.url);
```
Comment (inductor) [inline content/ja/examples/minikube/server.js]: This should be fine as-is, though you might have a valid reason to change this.

**Comment by inductor** [comment]
Okay thanks! I'll close this PR.

/close

**Comment by inductor** [comment]
@SupriyaSirbi Sorry, as you didn't mention https://github.com/kubernetes/website/pull/18532 I thought you sent the PR out of the blue. Thank you for your effort 🙇

**Pull request closed without merging**